settings.gradle

```diff
@@ -16,6 +16,14 @@
  * This is the root project descriptor for the Google Ads API Java library
  * gradle project.
  */
+pluginManagement {
+  repositories {
```

Comment:
Could you help me understand why this differs from the approach in [google-ads-java/google-ads-shadowjar/build.gradle](https://github.com/googleads/google-ads-java/blob/main/google-ads-shadowjar/build.gradle)?